openhtf/util/logs.py

```diff
@@ -77,54 +78,34 @@ def MyPhase(test, helper):
 import six
 
 
-DEFAULT_LEVEL = 'warning'
-DEFAULT_RECORD_VERBOSITY = 'debug'
-DEFAULT_LOGFILE_LEVEL = 'warning'
-QUIET = False
-LOGFILE = None
-
-LEVEL_CHOICES = ['debug', 'info', 'warning', 'error', 'critical']
 ARG_PARSER = argv.ModuleParser()
-ARG_PARSER.add_argument(
-    '--verbosity', default=DEFAULT_LEVEL, choices=LEVEL_CHOICES,
-    action=argv.StoreInModule, target='%s.DEFAULT_LEVEL' % __name__,
-    help='Console log verbosity level (stderr).')
-ARG_PARSER.add_argument(
-    '--test-record-verbosity', default=DEFAULT_RECORD_VERBOSITY, choices=LEVEL_CHOICES,
-    action=argv.StoreInModule, target='%s.DEFAULT_RECORD_VERBOSITY' % __name__,
-    help='Console log verbosity level for test record (not framework logger).')
-ARG_PARSER.add_argument(
-    '--quiet', action=argv.StoreInModule, target='%s.QUIET' % __name__,
-    proxy=argparse._StoreTrueAction, help="Don't output logs to stderr.")
-ARG_PARSER.add_argument(
-    '--log-file', action=argv.StoreInModule, target='%s.LOGFILE' % __name__,
-    help='Filename to output logs to, if any.')
-ARG_PARSER.add_argument(
-    '--log-file-level', default=DEFAULT_LOGFILE_LEVEL, choices=LEVEL_CHOICES,
-    action=argv.StoreInModule, target='%s.DEFAULT_LOGFILE_LEVEL' % __name__,
-    help='Logging verbosity level for log file output.')
+ARG_PARSER.add_argument('-v', action='append_const', const='v')
 
 LOGGER_PREFIX = 'openhtf'
-RECORD_LOGGER = '.'.join((LOGGER_PREFIX, 'test_record'))
+RECORD_LOGGER_PREFIX = '.'.join((LOGGER_PREFIX, 'test_record'))
 
 _LOGONCE_SEEN = set()
 
+
 LogRecord = collections.namedtuple(
     'LogRecord', 'level logger_name source lineno timestamp_millis message')
 
 
 def get_record_logger_for(test_uid):
-  return logging.getLogger('.'.join((RECORD_LOGGER, test_uid)))
+  return logging.getLogger('.'.join((RECORD_LOGGER_PREFIX, test_uid)))
 
 
 def initialize_record_logger(test_uid, test_record, notify_update):
+  htf_logger = logging.getLogger(LOGGER_PREFIX)
+  htf_logger.propagate = False
```

Comment:
Should this also occur when the logger gets the CLI handler attached in configure_cli_logging?

Reply:
Ah, that's a good idea.